platform/android/app/src/main/kotlin/coop/polypoly/polypod/homescreen/HomeScreenFragment.kt

```diff
@@ -0,0 +1,775 @@
+package coop.polypoly.polypod.homescreen
```

Comment:
Wow, 700+ lines...

Reply:
Yeah, I have to pull out more code form this file; Already did with `styles`, `layout`.